flash/text/classification/model.py

```diff
@@ -95,17 +95,10 @@ def backbone(self):
         return self.model.base_model
 
     def forward(self, batch: Dict[str, torch.Tensor]):
-        return self.model(input_ids=batch.get("input_ids", None), attention_mask=batch.get("attention_mask", None))
-
-    def to_loss_format(self, x) -> torch.Tensor:
-        if isinstance(x, (SequenceClassifierOutput, Seq2SeqSequenceClassifierOutput)):
-            x = x.logits
-        return super().to_loss_format(x)
-
-    def to_metrics_format(self, x) -> torch.Tensor:
-        if isinstance(x, (SequenceClassifierOutput, Seq2SeqSequenceClassifierOutput)):
-            x = x.logits
-        return super().to_metrics_format(x)
```

Comment:
Just for my own understanding, why are these not needed anymore?

Reply:
Just because we can do the conversion in the forward pass insteadand that would be equivalent to having it in both hooks :smiley: